.github/workflows/storybook-chromatic-branch.yml

```diff
@@ -0,0 +1,40 @@
+name: QA
+
+on:
+  pull_request:
+    types: [edited, labeled]
```

Comment:
Not sure what's expected, are you missing opened and synchronized events?

Reply:
I want to run this workflow only when label is added to PR - I took those from `e2e.yml` workflow